wolox-books/src/views/SignUp.vue

```diff
@@ -62,16 +79,23 @@ export default {
     margin: auto;
   }
 
+  .error {
+    color: $froly;
+    font-size: 11px;
+    margin: auto;
+    width: 250px;
+  }
+
   .input {
     border-radius: 5px;
     height: 25px;
-    margin: 2px auto 12px;
+    margin: 2px auto 2px;
```

Comment:
Why are you using a margin? :thinking: Maybe you're needing a padding.
In case it's a margin what you're looking for, you can do `margin: 2px auto;`.